static/js/containers/pages/CheckoutPage.js

```diff
@@ -219,34 +218,81 @@ export class CheckoutPage extends React.Component<Props, State> {
     )
 
     return (
-      <div className="checkout-page">
-        {this.renderBasketItem(item)}
-        <div className="row price-row">Price {formatPrice(item.price)}</div>
-        {coupon ? (
-          <div className="row discount-row">
-            Discount applied {formatPrice(calculateDiscount(item, coupon))}
+      <div className="checkout-page container">
+        <div className="row header">
+          <div className="col-12">
+            <div className="page-title">Checkout</div>
+            <div className="purchase-text">
+              You are about to purchase the following:
+            </div>
+            <div className="item-type">
+              {item.type === "program" ? "Program" : "Courses"}
```

Comment:
if the user can only purchase one course at a time, should this be `Course` instead of `Courses`?

Reply:
That change makes sense to me, I'll make the change. cc @abdulkdawson